uber/models.py

```diff
@@ -630,9 +630,22 @@ def everything(self, location=None):
                                         .order_by(Attendee.full_name).all()
                          if c.AT_THE_CON or not location or int(location) in a.assigned_depts_ints]
             for job in jobs:
-                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted]
+                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted_in(job.location)]
             return jobs, shifts, attendees
 
+        def staffers_by_job_options(self, job):
+            """
+            Format staffers_by_job() for use with the {% options %} template decorator
+            """
+            return [(a.id, a.full_name) for a in self.staffers_by_job(job)]
+
```

Comment:
agreed on this stuff, will do.

one interesting consequence of making this  a property on Job is that the difference between this and ```.available_staffers``` is so minimal that I might take the time to combine those into one thing.  more info below.